domains/crazo7924.json

```diff
@@ -7,7 +7,12 @@
         },
     
         "record": {
-            "TXT": "zoho-verification=zb09491787.zmverify.zoho.in"
+            "TXT": ["zoho-verification=zb09491787.zmverify.zoho.in", "_github-pages-challenge-crazo7924.crazo7924.is-a.dev=09f923d8e93c7000208a6202541915"],
```

Comment:
This is not how github pages verification works, you need to create another file named `_github-pages-challenge-crazo7924.crazo7924` and put only `09f923d8e93c7000208a6202541915` as TXT record there

see https://is-a.dev/docs/github-pages